README.markdown

```diff
@@ -323,7 +323,7 @@ load-nvmrc() {
     local nvmrc_node_version=$(nvm version "$(cat "${nvmrc_path}")")
 
     if [ "$nvmrc_node_version" != "N/A" ] && [ "$nvmrc_node_version" != "$node_version" ]; then
-      nvm use 
+      nvm install
```

Comment:
This will result in extra terminal output when it's already installed - although perhaps that's not a concern.